pkg/util/helpers.go

```diff
@@ -154,3 +155,22 @@ func IsExposeServiceValid(serviceType string) bool {
 	}
 	return false
 }
+
+// IsBlackDuckVersionSupportMultipleInstance returns whether it supports multiple instance in a single namespace
+func IsBlackDuckVersionSupportMultipleInstance(version string) (bool, error) {
+	return isYearAndMonthGreaterThanEqualTo(version, 2019, time.August)
+}
+
+func isYearAndMonthGreaterThanEqualTo(version string, year int, month time.Month) (bool, error) {
```

Comment:
Should this be named `isYearAndMonthGreaterOrEqualTo` instead?

Reply:
done